src/promptflow-azure/promptflow/azure/_cli/entry.py

```diff
@@ -123,24 +115,8 @@ def main():
     """Entrance of pf CLI."""
     command_args = sys.argv[1:]
     if len(command_args) == 1 and command_args[0] == "version":
-        version_dict = {"promptflow": get_promptflow_sdk_version()}
-        # check tracing version
-        version_tracing = get_promptflow_tracing_version()
-        if version_tracing:
-            version_dict["promptflow-tracing"] = version_tracing
-        # check azure version
-        version_azure = get_promptflow_azure_version()
-        if version_azure:
-            version_dict["promptflow-azure"] = version_azure
-        # check core version
-        version_core = get_promptflow_core_version()
-        if version_core:
-            version_dict["promptflow-core"] = version_core
-        # check devkit version
-        version_devkit = get_promptflow_devkit_version()
-        if version_devkit:
-            version_dict["promptflow-devkit"] = version_devkit
-        return json.dumps(version_dict, ensure_ascii=False, indent=2, sort_keys=True, separators=(",", ": ")) + "\n"
+        print_promptflow_version_dict_string(with_azure=True, ignore_none=True)
+        return
```

Comment:
Why do we have two places to print pf version? Here and line 42. What's the diff between this `print_promptflow_version_dict_string` and `print_pf_version`?

Reply:
I think no difference, only is the print format.

Reply:
extract similar logic to one function